src/subcommands/create_rsa_key.rs

```diff
@@ -27,6 +27,10 @@ pub struct CreateRsaKey {
     #[structopt(short = 's', long = "for-signing")]
     is_for_signing: bool,
 
+    /// Supply this flag to create a signing key with PSS scheme and SHA-256 hash algorithm.
```

Comment:
You should probably make clear both in the code documentation and in the CLI documentation what the "priority" between these two flags is (this one and the previous).

Reply:
Have added comments in the code and also in the CLI about this.